### DISCUSSION TITLE
Silent thermostat ?

### DISCUSSION THREAD

**Discussion by JacekKac** [Koenkk/zigbee2mqtt#17459] [Devices]
any recommedations? Right now i am using TS0601/ _TZE200_cwnjrr72 and it is to loud for me especially in bedrom

**Comment by lux73** [comment]
[Danfoss Ally](https://www.danfoss.com/en/products/dhs/smart-heating/smart-heating/danfoss-ally/#tab-overview) - unbelievable silent, really!

but they are unbelievable expensive ^^ but working remarkable nice
